Regression on payload validations on POST `/v2/evc/`

Today when we've fixed the docker build, I've retried to re-run the tests, noticed some failures (we'll get the rest of the list on tonight's execution), but this one here for instance, fails with an unexpected client side 400 error:

```
collected 189 items / 188 deselected
tests/test_e2e_10_mef_eline.py F                                         [100%]
=================================== FAILURES ===================================
____ TestE2EMefEline.test_050_on_primary_path_fail_should_migrate_to_backup ____
self = <tests.test_e2e_10_mef_eline.TestE2EMefEline instance at 0x7f0095632a28>
    def test_050_on_primary_path_fail_should_migrate_to_backup(self)
```

```
{
    "code": 400,
    "description": "Additional properties are not allowed ('backup_path' was unexpected) for field .",
    "name": "Bad Request"
}
```

You can reproduce the issue with this payload for example:

```
{
    "name": "my evc1",
    "enabled": true,
    "uni_a": {
        "interface_id": "00:00:00:00:00:00:00:01:1"
    },
    "uni_z": {
        "interface_id": "00:00:00:00:00:00:00:03:1"
    },
    "primary_path": [
        {
            "endpoint_a": {
                "id": "00:00:00:00:00:00:00:01:2"
            },
            "endpoint_b": {
                "id": "00:00:00:00:00:00:00:02:2"
            }
        },
        {
            "endpoint_a": {
                "id": "00:00:00:00:00:00:00:02:3"
            },
            "endpoint_b": {
                "id": "00:00:00:00:00:00:00:03:2"
            }
        }
    ],
    "backup_path": [
        {
            "endpoint_a": {
                "id": "00:00:00:00:00:00:00:01:3"
            },
            "endpoint_b": {
                "id": "00:00:00:00:00:00:00:03:3"
            }
        }
    ]
}
```

It looks like it's something related to the current [`openapi.yml` spec and the `NewCircuit` schema with the `backup_path` key](https://github.com/kytos-ng/mef_eline/blob/master/openapi.yml#L315-L319)

--- a/openapi.yml
+++ b/openapi.yml
@@ -307,15 +307,15 @@ components:
         primary_links:
           type: array
           items:
-            $ref: '#/components/schemas/Path'
+            $ref: '#/components/schemas/NewLink'
         backup_links:
           type: array
           items:
-            $ref: '#/components/schemas/Path'
+            $ref: '#/components/schemas/NewLink'
         primary_path:
           type: array
           items:
-            $ref: '#/components/schemas/Path'
+            $ref: '#/components/schemas/NewLink'
         dynamic_backup_path:
           type: boolean
         circuit_scheduler:
@@ -327,7 +327,26 @@ components:
           format: int64
         enabled:
           type: boolean
-
+    NewLink:
+      type: object
+      required:
+        - endpoint_a
+        - endpoint_b
+      properties:
+        endpoint_a:
+          $ref: '#/components/schemas/NewEndpoint'
+        endpoint_b:
+          $ref: '#/components/schemas/NewEndpoint'
+    NewEndpoint:
+      type: object
+      required:
+        - id
+      properties:
+        id:
+          type: string
+          minLength: 25
+          maxLength: 30
+          pattern: '^([0-9A-Fa-f]{2}[:-]){8}(\d{1,5})$'          
     Endpoint: # Can be referenced via '#/components/schemas/Endpoint'
       type: object
       required:

--- a/openapi.yml
+++ b/openapi.yml
@@ -316,6 +316,10 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/NewLink'
+        backup_path:
+          type: array
+          items:
+            $ref: '#/components/schemas/NewLink'
         dynamic_backup_path:
           type: boolean
         circuit_scheduler:

--- a/utils.py
+++ b/utils.py
@@ -79,7 +79,9 @@ def validate(spec):
                     }
                     log.debug("error response: %s", error_log)
                     error_response = f"{schema_errors.message} for field"
-                    error_response += f" {'/'.join(schema_errors.path)}."
+                    error_response += (
+                        f" {'/'.join(map(str,schema_errors.path))}."
+                    )
                 else:
                     error_response = (
                         "The request body mimetype is not application/json."

